packages/@coorpacademy-components/package.json

```diff
@@ -20,6 +20,7 @@
     "build:es": "npm run build:es:babel && npm run build:es:types",
     "build": "concurrently \"npm run build:commonjs\" \"npm run build:es\"",
     "build:watch": "npm run build:es -- -w --verbose",
+    "build:stats": "webpack --profile --json > dist/stats.json",
```

Comment:
should we auto update the stats on every new deployment? @AdrieanKhisbe

Reply:
I'm not sure. so far the command open a web browser on 8888 with the treemap. (and the output is just used locally) Also, because of that terminal keeps focus which might cause trouble in CI context